addons/mail/static/src/components/attachment_viewer/attachment_viewer.xml

```diff
@@ -21,12 +21,15 @@
                     </div>
                 </t>
                 <div class="o_AttachmentViewer_headerItem o_AttachmentViewer_name">
-                    <t t-esc="attachmentViewer.attachment.displayName"/>
+                    <span class="o_AttachmentViewer_nameText" t-esc="attachmentViewer.attachment.displayName"/>
```

Comment:
Just to clarify why I'm doing this: apparently `text-overflow: ellipsis` does not work with `flex` in some cases.
The workaround consists of relying on `display: inline` of `<span>` just for the text, and put `text-overflow: ellipsis` on the span itself only.

The container is kept for flex, and the expected adaptive behaviour requires `min-width: 0` workaround.

Reply:
Ok thanks for the details, looks like a known "issue" indeed: https://css-tricks.com/flexbox-truncated-text/
This page says our case should have been fine, but I'm not against the nested version as done here.